<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EntryDialog</class>
 <widget class="QDialog" name="EntryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Urszula Chmielewska - Add Recipe</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>609</width>
     <height>548</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
color: #FFC700;
font-size: 15px;</string>
            </property>
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="titleRecipe">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>-1</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 0.5px;
border-color: #54565B;
background-color: #FFFFFF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
color: #FFC700;
font-size: 15px;</string>
            </property>
            <property name="text">
             <string>Recipe</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="txtRecipe">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>-1</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 0.5px;
border-color: #54565B;
background-color: #FFFFFF</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="autoFillBackground">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
color: #FFC700;
font-size: 15px;</string>
            </property>
            <property name="text">
             <string>Ingredients</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="ingredientsList">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>-1</pointsize>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">font: &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 0.5px;
border-color: #54565B;
background-color: #FFFFFF</string>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="btnOk">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 13px;
background-color: #FFCB05;
color: white;</string>
          </property>
          <property name="text">
           <string>Ok</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnCancel">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 13px;
background-color: #FF665A;
color: #FFFFFF;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 12px;
font: bold;
color: white;</string>
        </property>
        <property name="text">
         <string>Add ingredient</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Gill Sans MT</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Product</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="product">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 1px;
border-color: #FFC700;
background-color: #FFFFFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Gill Sans MT</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Amount</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="amount">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 1px;
border-color: #FFC700;
background-color: #FFFFFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Gill Sans MT</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;</string>
          </property>
          <property name="text">
           <string>Unit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="unit">
          <property name="styleSheet">
           <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 11px;
border-style: solid;
border-width: 1px;
border-color: #FFC700;
background-color: #FFFFFF</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnAdd">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 12px;
background-color: #363636;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnRemove">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 8pt &quot;Gill Sans MT&quot;;
font-size: 12px;
background-color: #363636;
color: #FFFFFF;</string>
        </property>
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
